config/config.go

```diff
@@ -100,6 +104,17 @@ type Pubsub struct {
 	Publisher *pubsub.Client
 }
 
+// Kinesis config for the AWS Kinesis
```

Comment:
We should add a comment here that is optional for the one pursuing AWS kinesis integration.